fud/fud/config.py

```diff
@@ -26,6 +26,12 @@
             "file_extensions": [".futil"],
             "flags": None,
         },
+        "interpreter": {
+            "exec": "./target/debug/interp",
+            "flags": None,
+            "data": None,
+            "round_float_to_fixed": True,
+        },
```

Comment:
Add a new check for `fud check` to warn users if this has not been built.

Reply:
Fud check already prints a warning if the interpreter path doesn't point to anything. Were you looking for something else?

Reply:
I think that's probably all that's necessary!

Reply:
Oh, you’re right. This sounds good.